onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -858,6 +870,27 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
     input_names_str_[i] = m.GetInputName(i);
     input_names_[i] = input_names_str_[i].c_str();
   }
+
+  if (use_device_mem) {
+    Ort::MemoryInfo memory_info = Ort::MemoryInfo("OpenVINO_RT_NPU", OrtArenaAllocator, 0, OrtMemTypeCPUOutput);
```

Comment:
Although `use_device_mem` is currently only set for OVEP, this code should probably be EP-agnostic. In fact, this could be used by other EPs in the future to set their own custom allocators.

What do you think of converting the field `use_device_mem` to `const char* device_memory_name = nullptr`. Then, when you're parsing the options for OVEP, you set `device_memory_name` to `"OpenVINO_RT_NPU"`. This makes this usable by other EPs in the future.

Reply:
to be implemented in followup